tests/system/action/poll/test_create.py

```diff
@@ -102,3 +107,318 @@ def test_create_three_options(self) -> None:
         assert option_4.get("used_as_global_option_in_poll_id") == 1
         assert option_4.get("meeting_id") == 112
         assert option_4.get("weight") == 1
+
+
+class CreatePollTestCase(BaseActionTestCase):
```

Comment:
Same here, whts the difference between `CreatePollTestCase` and `PollCreateTest`. By the way, I think you can remove the suffixes `TestCase` and `Text`. IMO they do not deliver much infomation...

Reply:
Removed the "TestCase" and "Test" from the classname.

Reply:
Merged the tests into one class. Removed some surplus tests.